code/sound/ffmpeg/FFmpegWaveFile.cpp

```diff
@@ -236,6 +236,64 @@ bool FFmpegWaveFile::Open(const char* pszFilename, bool keep_ext)
 	return true;
 }
 
+// OpenMem
+bool FFmpegWaveFile::OpenMem(const uint8_t* snddata, size_t snd_len)
+{
+	using namespace libs::ffmpeg;
+
+
+	try {
+		m_ctx    = FFmpegContext::createContextMem(snddata, snd_len);
```

Comment:
Again duplicated code. The context creation is the only different thing and can be extracted.